db/schema.rb

```diff
@@ -60,6 +60,8 @@
   end
 
   create_table "permit_applications", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.integer "permit_type", default: 0
+    t.integer "building_type", default: 0
```

Comment:
Might need to drop database and remigrate. I think keith removed these fields, so it might be getting picked up from existing db fields